src/common/grpc.rs

```diff
@@ -46,6 +49,33 @@ impl GrpcBody {
     }
 }
 
+impl GrpcResponse for GrpcBody {
+    fn put_data(&mut self, data: Bytes) {
+        let _ = self.data.insert(data);
```

Comment:
`insert` replaces the old value in an Option with a new one returning if if it exists. In this flow because we may reuse older grpc bodies for new requests we can safely forgo whatever data there is